index.php

```diff
@@ -119,6 +119,11 @@
     unset($_GET['updated']);
 }
 
+//handle genre filter
+if (isset($_POST['selectGenre'])) {
+    $allRecords = $recordModel->getAllRecords($genreFilterID);
+}
```

Comment:
If you have a genre filter set, you end up querying the database twice. Once initially to get all records, and then again this time - and the results of the first query end up being useless. 

You should update this so it either gets them with a genreFilter, or gets all of them. Not both